### PR TITLE
Header link to contrib organization

### PR DESCRIPTION
**Pull request by pavolloffay**
@yurishkuro @bensigelman What do you think about this? I think it would be good to have a link to the contrib organization in menu too.

### CODE REVIEW

**Comment by bensigelman** [comment]
@pavolloffay great idea, yes. There is also github.com/opentracing/contrib, but IMO we should get rid of that, or maybe move it to github.com/opentracing-contrib/meta or something.

File: _data/language.yaml [hunk 1]
```diff
@@ -23,3 +23,7 @@ array:
   - name: C++
     url: //github.com/opentracing/opentracing-cpp
     icon: ''
+
+  - name: Contributions
+    url: //github.com/opentracing-contrib
```
Comment (bensigelman) [inline _data/language.yaml]: I have created https://github.com/opentracing-contrib/meta ... maybe link there?

Comment (pavolloffay) [inline _data/language.yaml]: :+1:  it works for me. Maybe update `meta/README` would be great.

Comment (bensigelman) [inline _data/language.yaml]: @pavolloffay friendly ping about this PR...

Comment (pavolloffay) [inline _data/language.yaml]: @wu-sheng yes, it adds `Contributors` under `C++`

Comment (wu-sheng) [inline _data/language.yaml]: But the title is `Libraries available in 6 languages`, and `Contributions` under this, seems strange? @pavolloffay .
`Contributions` contains something about using OT-x-languages, not add a new supported languages.

I prefer to add `Contributions` at the homepage top menu, maybe after `BLOG`.

Comment (pavolloffay) [inline _data/language.yaml]: @wu-sheng add `Contributors`after `Blog` seems fine to me.  

@bensigelman thoughts? Do you agree?

Comment (wu-sheng) [inline _data/language.yaml]: @pavolloffay , also, you should change the url to `https://github.com/opentracing-contrib/meta`, as Ben mentioned before.

Comment (bensigelman) [inline _data/language.yaml]: @pavolloffay this looks good to me after it points to https://github.com/opentracing-contrib/meta as @wu-sheng mentioned. Thanks.

Comment (wu-sheng) [inline _data/language.yaml]: @pavolloffay , after you change the link's position, and link itself. This PR will be ready to be merged. 😊

**Comment by pavolloffay** [comment]
@wu-sheng @bensigelman PR updated

**Comment by wu-sheng** [comment]
@pavolloffay, Thank you.

**Comment by bensigelman** [comment]
@pavolloffay this link doesn't appear in mobile... I'll try to fix this later on, should be easy enough.

**Comment by reimertz** [comment]
@bhs let me know if you need support.

**Comment by bensigelman** [comment]
@reimertz if you want to do it, please go ahead :)

**Comment by bensigelman** [comment]
See #181.

**Pull request merged**